sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterTraceExtensions.cs

```diff
@@ -59,7 +64,14 @@ public static class AzureMonitorExporterTraceExtensions
                     configure(exporterOptions);
                 }
 
-                return new BatchActivityExportProcessor(new AzureMonitorTraceExporter(exporterOptions, credential));
+                var standardMetricReader = new PeriodicExportingMetricReader(new AzureMonitorMetricExporter(exporterOptions, credential))
```

Comment:
What is the idea behind creating a new `PeriodicExportingMetricReader`? 
Is the plan to send data even if customer does not have metric exporter initialized?

Reply:
Yes - standard metric should work independent of customer's metric exporter

Reply:
removed additional periodicExportingMetricReader